### DISCUSSION TITLE
Widget not showing properly

### DISCUSSION THREAD

**Discussion by kulbhushanchand** [mmistakes/minimal-mistakes#2873] [Q&A]
On my [webpage](https://kulbhushanchand.github.io/html-widgets-test/), I have included a html-widget to show interactive table. The **expected output** is as shown in figure below (also working demo at - https://rstudio.github.io/DT/).

**Issue -** The table is not showing with the minimal-mistakes theme.  **However, the problem is resolved if**, I disable the theme javascript file `main.min.js`. As per this [stackoverflow answer](https://stackoverflow.com/questions/31227844/typeerror-datatable-is-not-a-function), the problem is with the loading of `jQuery` twice or with the loading order of the two js files `jQuery` and `jQuery DataTables`. (Both the theme and html-widget uses `jQuery ver 3.5.1`)

**What can be the possible solution for this edge case?**

![image](https://user-images.githubusercontent.com/17742733/110810569-ed77c500-82ab-11eb-89a1-17f4c378d845.png)

**Comment by iBug** [comment]
Load your jQuery plugin with [`site.after_footer_scripts`][1]. Your plugin will be loaded *after* jQuery is available.

  [1]:https://github.com/mmistakes/minimal-mistakes/blob/3d3cb58a2efa2c8b1ed4d9f93f48419e9e0aedec/_includes/scripts.html#L24

**Reply by kulbhushanchand** [reply]
I tried using `after_footer_scripts:` in YAML (manually), but no effect. Also, the scripts are programmatically included in the end of the md file, so no luck. 
The only solution so far is to disable the theme's `js` file.

**Reply by kulbhushanchand** [reply]
Just realized my mistake that I was adding `after_footer_scripts:` in YAML of md file, instead of inside the `_config.yml`. The downside of this approach is that scripts gets loaded for each page irrespective of included widget.
